pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -208,6 +208,7 @@ public void setNumSegmentsMatched(long numSegmentsMatched) {
   }
 
   @JsonProperty("numConsumingSegmentsQueried")
+  @Override
   public long getNumConsumingSegmentsQueried() {
```

Comment:
Not related to this PR, but can you add override this to getNumSegmentsQueried, getNumSegmentsProcessed, getNumSegmentsMatched as well?